docs/js/signature_pad.umd.js

```diff
@@ -358,7 +358,19 @@
       };
       SignaturePad.prototype._createPoint = function (x, y) {
           var rect = this.canvas.getBoundingClientRect();
-          return new Point(x - rect.left, y - rect.top, new Date().getTime());
+          var positionOnCanvasElement = {
+              x: (x - rect.left),
+              y: (y - rect.top)
+          };
+          var ratio = {
```

Comment:
This object will never change, correct?  It would be more efficient if it wasn't calculated on every call to_createPoint(...).

Reply:
It's quiet a long time since I implemented this one. So I'm not :100:ly sure to be honest. I fear that in some edge cases window resize or zoon could alter `clientWidth` or `clientHeight` of the `canvas`.

I don't think the performance difference is significant. We execute `getBoundingClientRect()` on `canvas` anyways. So the browser would calculate styles and do the layout even if the object would be cached.

I tend to keep it for now and refactor to a cached solution later. Don't want to have too much risky changes in one pull request. Especially as there are no tests if I recall correctly.